test/integration/WovnIndexSampleApacheTest.php

```diff
@@ -0,0 +1,225 @@
+<?php
+namespace Wovnio\Wovnphp\Tests\Integration;
+
+use Wovnio\Test\Helpers\Utils;
+
+class WovnIndexSampleApacheTest extends \PHPUnit_Framework_TestCase
+{
+    protected function setUp()
+    {
+        $this->sourceDir  = dirname(__FILE__) . '/../..';
+        $this->docRoot    = dirname(__FILE__) . '/../docroot';
+        $this->installDir = $this->docRoot . '/WOVN.php';
+
+        mkdir($this->installDir);
+        exec(sprintf('cp -rf %s %s', $this->sourceDir . '/src', $this->installDir . '/src'));
+
+        copy($this->sourceDir . '/wovn_index_sample.php', $this->docRoot . '/wovn_index.php');
+        copy($this->sourceDir . '/htaccess_sample', $this->docRoot . '/.htaccess');
+
+        mkdir($this->docRoot . '/v0');
+        copy($this->sourceDir . '/test/fixtures/integration/v0/translation', $this->docRoot . '/v0/translation');
+
+        chdir($this->docRoot);
+    }
+
+    protected function tearDown()
+    {
+        Utils::cleanUpDirectory($this->docRoot);
```

Comment:
I prefer cleaning up at the first of setUp.
Because If some test fails, after tests affect the failed tests.